src/pages/index.js

```diff
@@ -294,12 +294,12 @@ class Home extends Component {
                 }}>
                 <CtaItem>
                   <ButtonLink to="/docs/getting-started.html" type="primary">
-                    Get Started
+                  شروع کریں
                   </ButtonLink>
                 </CtaItem>
                 <CtaItem>
                   <ButtonLink to="/tutorial/tutorial.html" type="secondary">
-                    Take the Tutorial
+                  سبق لے لو
```

Comment:
I think it is better as 
```
سبق حاصل کریں
```

Reply:
ok I will make changes and will create a new issue